### PR TITLE
Arrangement_2: Add Forward Declaration

### PR DESCRIPTION
**Pull request by afabri**
## Summary of Changes

Also inside a class scope we have to add forward declarations.

## Release Management

* Affected package(s):  Arrangement_2 and what uses it
* Issue(s) solved (if any):  VC2017 preview

### CODE REVIEW

**Comment by lrineau** [comment]
This PR should be for 4.12-branch.

**Comment by sloriot** [comment]
Tested in CGAL-4.13-Ic-11

File: Installation/include/CGAL/disable_warnings.h [hunk 1]
```diff
@@ -34,6 +34,7 @@
 #  pragma warning(disable: 4714) // function marked as __forceinline not inlined
 #  pragma warning(disable: 4800) //  forcing value to bool 'true' or 'false' (performance warning)
 #  pragma warning(disable: 4913) // user defined binary operator ',' exists but no overload could convert all operands, default built-in binary operator ',' used
+#  pragma warning(disable: 4834) // discarding return value of function with 'nodiscard' attribute
```
Comment (lrineau) [inline Installation/include/CGAL/disable_warnings.h]: That list of disabled warnings is frightening. The description of most of them look like real bugs in the code.

**Comment by afabri** [comment]
comment what you want to comment and see what it does in the testsuite.

**Pull request merged**